### DISCUSSION TITLE
color gradients

### DISCUSSION THREAD

**Discussion by charleschege** [emilk/egui#783] [Q&A]
Is there a way to do color gradients in egui like 
![5a038e76e658daff1c50256879bc84c8](https://user-images.githubusercontent.com/33346042/136738698-4b5d39a8-8b40-4621-8504-87f13942345d.jpg)

**Comment by kud1ing** [comment]
AFAIK not directly. The Color Test from the [Demo](https://www.egui.rs) generates [Mesh](https://docs.rs/egui/latest/egui/struct.Mesh.html)es for the gradients.
